chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -553,64 +554,83 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const prepareEndChatRelay = () => prepareEndChat(props, chatSDK, state, dispatch, setAdapter, setWebChatStyles, adapter, uwid.current);
 
+    const chatWidgetDraggableConfig = {
+        elementId: widgetElementId,
+        channel: props.controlProps?.widgetInstanceId ?? "lcw",
+        disable: props.draggableChatWidgetProps?.disable === true?? false // Draggable by default, unless explicitly disabled
```

Comment:
"// Draggable by default, unless explicitly disabled" is this by PM requirement. I would think the opposite - disable by default. Otherwise it might break some customers' UI scenarios

Reply:
By changing `disable` to `enable`

Reply:
It will be enabled by default after offline discussion